spec/context/dimension.fmf

```diff
@@ -0,0 +1,50 @@
+summary: Supported context dimensions
+
+story:
+    As a tester I want to clearly specify dimensions, such as
+    'distro' or 'trigger', so that I can well describe the
+    context in which a test is running.
+
+description: |
+    The following dimensions are supported:
+
+    product
+        The target product name for which the application is
+        intended (fedora, fedora-33, centos, centos-8,
+        centos-84, rhel, rhdts, rhscl).
+
+    distro
+        The operating system distribution on which the application
+        runs (fedora, fedora-33, centos, centos-8, centos-8.4,
+        rhel).
+
+    variant
```

Comment:
Should we mention this is only for RHEL7 / CentOS 7?

Reply:
Hm, not sure - Fedora has Silverblue, CoreOs, Server, Workstation ....

Reply:
Added Fedora examples as well.